qa/rpc-tests/sporks.py

```diff
@@ -69,6 +70,16 @@ def run_test(self):
         assert(not self.get_test_spork_state(self.nodes[0]))
         assert(not self.get_test_spork_state(self.nodes[1]))
 
+        # Force finish mnsync node as otherwise it will never send out headers to other peers
+        while True:
+            s = self.nodes[1].mnsync('next')
+            if s == 'sync updated to MASTERNODE_SYNC_FINISHED':
+                break
+            sleep(0.1)
```

Comment:
Could replace this whole `while` thing with `wait_to_sync(self.nodes[1], True)`.